zipline/src/commonMain/kotlin/app/cash/zipline/internal/bridge/inbound.kt

```diff
@@ -26,7 +26,7 @@ import okio.Buffer
  */
 @PublishedApi
 internal abstract class InboundService<T : Any>(
-  internal val serializersModule: SerializersModule
```

Comment:
Were these made non-internal for dev / testing only?

Reply:
Actually they must be internal so that subclasses can override them.